nautobot/extras/urls.py

```diff
@@ -156,6 +157,18 @@
         views.CustomFieldDeleteView.as_view(),
         name="customfield_delete",
     ),
+    path(
+        "custom-fields/<uuid:pk>/changelog/",
+        views.ObjectChangeLogView.as_view(),
+        name="customfield_changelog",
+        kwargs={"model": CustomField},
+    ),
```

Comment:
This shouldn't be needed - instead, it looks like the right thing to do would be override `get_changelog_url` on `CustomFieldView` so that it does the lookup by name as per the path below.